eng/test-job.yml

```diff
@@ -203,6 +203,31 @@ jobs:
           - jitstress2_tiered
           - zapdisable
           - tailcallstress
+        ${{ if eq(parameters.testGroup, 'outerloop-jitstressisas') }}:
+          scenarios:
+          - jitstressisas_incompletehwintrinsic
+          - jitstressisas_nohwintrinsic
+          - jitstressisas_nohwintrinsic_nosimd
+          - jitstressisas_nosimd
+        ${{ if and(eq(parameters.testGroup, 'outerloop-jitstressisas'), or(eq(parameters.archType, 'x64'), eq(parameters.archType, 'x86'))) }}:
+          scenarios:
+          - jitstressisas_noaes
+          - jitstressisas_noavx
+          - jitstressisas_noavx2
+          - jitstressisas_nobmi1
+          - jitstressisas_nobmi2
+          - jitstressisas_nofma
+          - jitstressisas_nohwintrinsicx86
+          - jitstressisas_nolzcnt
+          - jitstressisas_nopclmulqdq
+          - jitstressisas_nopopcnt
+          - jitstressisas_nosse
+          - jitstressisas_nosse2
+          - jitstressisas_nosse3
+          - jitstressisas_nosse3_4
+          - jitstressisas_nosse41
+          - jitstressisas_nosse42
+          - jitstressisas_nossse3
         ${{ if eq(parameters.testGroup, 'outerloop-jitstressregs') }}:
```

Comment:
As a note, it is likely worthwhile to add an x86/x64 specific `outerloop-jistressregs-noavx` scenario. AVX can impact register allocation scenarios because most SIMD instructions change from `ins op1, src` (where `op1` is both a dst and src) to `ins dst, src, src`. This applies to both the scalar SIMD instructions, like used for normal floating-point arithmetic and vector SIMD instructions.

Most other isa knobs don't have any significant impact other than the number of instructions emitted, so they may not be particularly interesting to combine with the other stress scenarios.

Reply:
CC. @CarolEidt

Reply:
Agree - that scenario would be very useful (and also agree that exploding to cover all the ISA configurations doesn't provide a lot of incremental value).

Reply:
I'll open a separate PR to cover that scenario, after this ones goes in.